flytekit/types/file/file.py

```diff
@@ -272,6 +272,10 @@ def to_literal(
         if python_val is None:
             raise TypeTransformerFailedError("None value cannot be converted to a file.")
 
+        # Correctly handle `Annotated[FlyteFile, ...]` by extracting the origin type
+        if typing.get_origin(python_type) is typing.Annotated:
```

Comment:
import `Annotated` from `typing_extensions` so that we can use it in python 3.8.

Reply:
Fixed here and in test code in aa18baa6